components/form/form-element-mixin.js

```diff
@@ -227,6 +227,18 @@ export const FormElementMixin = superclass => class extends LocalizeCoreElement(
 		await this.updatedComplete;
 	}
 
+	reset() {
+		this.invalid = false;
+		this.validationError = null;
+		this._errors = [];
+
+		this.childErrors.forEach((_, ele) => {
+			if (!isCustomFormElement(ele)) return;
```

Comment:
A case (?) that wouldn't be covered is if the child error is a native element. I don't know if that's actually a thing we deal with.